vaadinfrontend/frontend/themes/datamanager/components/dialog.css

```diff
@@ -0,0 +1,57 @@
+.dialog {
+}
+
+.dialog::part(overlay) {
+  width: 66vw;
+}
+
+.dialog .confirm-button {
+  background-color: var(--lumo-primary-color);
+  color: var(--lumo-primary-contrast-color);
+  font-weight: 600;
+  min-width: calc(var(--lumo-button-size) * 2.5);
+}
+
+.dialog .confirm-button:hover::before {
+  opacity: 0.05;
+}
+
+.create-project-dialog {
+
+}
+
+.create-project-dialog .project-form-layout {
+  max-width: 60vw;
+  padding-bottom: 1em;
+}
+
+.create-project-dialog .code-and-title {
+  align-items: baseline;
+  width: 100%;
+}
+
+.create-project-dialog .code-and-title .code {
+  max-width: 20vw;
+}
+
+.create-project-dialog .code-and-title .title {
+  width: 100%;
+}
+
+.create-project-dialog .search {
+  max-width: 30vw;
+}
```

Comment:
I'd recommend to first replace all the vaadin components in the create-project-dialog with more basic div/span elements. Currently there is a lot of padding and margin done by the layouts used